spec/models/orphan_spec.rb

```diff
@@ -53,7 +44,7 @@
 
   it { is_expected.to_not allow_value(nil).for(:sponsored_by_another_org) }
 
-  #it { is_expected.to validate_numericality_of(:minor_siblings_count).only_integer.is_greater_than_or_equal_to(0) }
```

Comment:
Everything is passing. And if it weren't, we'd have to fix it and not comment it out :-)